runtime/j9vm/java9vmi.c

```diff
@@ -1777,3 +1777,11 @@ JVM_GetNanoTimeAdjustment(JNIEnv *env, jclass clazz, jlong offsetSeconds)
 	
 	return result;
 }
+
+#if J9VM_JCL_SE11
+void JNICALL
+JVM_BeforeHalt()
+{
+	/* To be implemented via https://github.com/eclipse/openj9/issues/1459 */
```

Comment:
There should be an Assert or something here if this is called.

Reply:
Actually this is being called indeed. I feel the implementation can be postponed in another issue to allow `Java 11 b04` build proceed first.